pkg/channel/fanout/fanout_message_handler.go

```diff
@@ -33,17 +35,32 @@ import (
 	"go.opencensus.io/trace"
 	"go.uber.org/zap"
 
+	eventingduckv1 "knative.dev/eventing/pkg/apis/duck/v1"
 	eventingduck "knative.dev/eventing/pkg/apis/duck/v1beta1"
 	"knative.dev/eventing/pkg/channel"
+	"knative.dev/eventing/pkg/kncloudevents"
 )
 
 const (
```

Comment:
Not your doing, but 15 minute timeout? :) I'll look what this is for. Just a note :)

Reply:
It seems too much :)

It's used here: https://github.com/knative/eventing/blob/d5ea26ffe3403a1325300446fc9b59cb4a94496c/pkg/channel/fanout/fanout_message_handler.go#L175-L193